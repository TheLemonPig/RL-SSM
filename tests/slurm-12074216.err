module: unloading 'graphviz/2.40.1'
module: loading 'graphviz/2.40.1'
module: unloading 'python/3.9.0'
module: loading 'python/3.9.0'
module: unloading 'git/2.29.2'
module: loading 'git/2.29.2'
WARNING (pytensor.tensor.blas): Using NumPy C-API based implementation for BLAS functions.
Multiprocess sampling (4 chains in 4 jobs)
CompoundStep
>Slice: [a]
>Slice: [b]
slurmstepd: error: *** JOB 12074216 ON node1702 CANCELLED AT 2023-11-15T14:10:33 ***
