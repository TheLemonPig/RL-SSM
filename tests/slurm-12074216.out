## SLURM PROLOG ###############################################################
##    Job ID : 12074216
##  Job Name : CalibrationStudy
##  Nodelist : node1702
##      CPUs : 1
##  Mem/Node : 4096 MB
## Directory : /oscar/home/jhewson/RL-SSM/tests
##   Job Started : Wed Nov 15 13:53:27 EST 2023
###############################################################################
Master process running on node1702.oscar.ccv.brown.edu
Directory is /users/jhewson/RL-SSM/tests
Starting execution at Wed Nov 15 13:53:28 EST 2023
Current PATH is /oscar/home/jhewson/RL-SSM/venv/bin:/gpfs/runtime/opt/intel/2017.0/bin:/gpfs/runtime/opt/python/2.7.12/bin:/gpfs/runtime/opt/matlab/R2017b/bin:/gpfs/runtime/opt/java/jdk-19.0.2/bin:/usr/lib/qt-3.3/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/usr/lpp/mmfs/bin:/usr/lpp/mmfs/sbin:/opt/ibutils/bin:/gpfs/runtime/bin:/opt/singularity/2.5.2/bin:/users/jhewson/bin
â–ˆ |----------------------------------------| 0.00% [0/16000 00:00<? Sampling 4 chains, 0 divergences] |--------------------------------| 0.01% [1/16000 01:56<517:29:07 Sampling 4 chains, 0 divergences] |--------------------------------| 0.01% [2/16000 02:23<319:04:28 Sampling 4 chains, 0 divergences] |--------------------------------| 0.02% [3/16000 02:29<220:44:14 Sampling 4 chains, 0 divergences] |--------------------------------| 0.03% [4/16000 02:31<168:43:52 Sampling 4 chains, 0 divergences] |--------------------------------| 0.03% [5/16000 02:35<138:02:33 Sampling 4 chains, 0 divergences] |--------------------------------| 0.04% [6/16000 02:50<126:00:34 Sampling 4 chains, 0 divergences] |--------------------------------| 0.04% [7/16000 02:56<112:00:21 Sampling 4 chains, 0 divergences] |--------------------------------| 0.05% [8/16000 03:01<100:37:26 Sampling 4 chains, 0 divergences] |--------------------------------| 0.06% [9/16000 03:02<89:51:17 Sampling 4 chains, 0 divergences] |--------------------------------| 0.06% [10/16000 03:16<87:15:49 Sampling 4 chains, 0 divergences] |--------------------------------| 0.07% [11/16000 03:25<82:57:10 Sampling 4 chains, 0 divergences] |--------------------------------| 0.07% [12/16000 03:38<80:44:59 Sampling 4 chains, 0 divergences] |--------------------------------| 0.08% [13/16000 03:39<74:51:04 Sampling 4 chains, 0 divergences] |--------------------------------| 0.09% [14/16000 03:48<72:34:04 Sampling 4 chains, 0 divergences] |--------------------------------| 0.09% [15/16000 03:56<70:00:59 Sampling 4 chains, 0 divergences] |--------------------------------| 0.10% [16/16000 04:07<68:37:06 Sampling 4 chains, 0 divergences] |--------------------------------| 0.11% [17/16000 04:30<70:40:52 Sampling 4 chains, 0 divergences] |--------------------------------| 0.11% [18/16000 04:31<67:02:01 Sampling 4 chains, 0 divergences] |--------------------------------| 0.12% [19/16000 04:39<65:17:45 Sampling 4 chains, 0 divergences] |--------------------------------| 0.12% [20/16000 04:47<63:42:03 Sampling 4 chains, 0 divergences] |--------------------------------| 0.13% [21/16000 04:59<63:22:27 Sampling 4 chains, 0 divergences] |--------------------------------| 0.14% [22/16000 05:03<61:14:51 Sampling 4 chains, 0 divergences] |--------------------------------| 0.14% [23/16000 05:05<58:57:56 Sampling 4 chains, 0 divergences] |--------------------------------| 0.15% [24/16000 05:23<59:53:43 Sampling 4 chains, 0 divergences] |--------------------------------| 0.16% [25/16000 05:36<59:40:23 Sampling 4 chains, 0 divergences] |--------------------------------| 0.16% [26/16000 05:46<59:11:24 Sampling 4 chains, 0 divergences] |--------------------------------| 0.17% [27/16000 05:56<58:36:56 Sampling 4 chains, 0 divergences] |--------------------------------| 0.17% [28/16000 06:15<59:27:14 Sampling 4 chains, 0 divergences] |--------------------------------| 0.18% [29/16000 06:15<57:26:02 Sampling 4 chains, 0 divergences] |--------------------------------| 0.19% [30/16000 06:39<59:05:26 Sampling 4 chains, 0 divergences] |--------------------------------| 0.19% [31/16000 06:48<58:28:45 Sampling 4 chains, 0 divergences] |--------------------------------| 0.20% [32/16000 06:50<56:50:44 Sampling 4 chains, 0 divergences] |--------------------------------| 0.21% [33/16000 06:54<55:41:17 Sampling 4 chains, 0 divergences] |--------------------------------| 0.21% [34/16000 07:00<54:50:03 Sampling 4 chains, 0 divergences] |--------------------------------| 0.22% [35/16000 07:24<56:19:17 Sampling 4 chains, 0 divergences] |--------------------------------| 0.23% [36/16000 07:29<55:20:36 Sampling 4 chains, 0 divergences] |--------------------------------| 0.23% [37/16000 07:32<54:15:09 Sampling 4 chains, 0 divergences] |--------------------------------| 0.24% [38/16000 07:33<52:57:57 Sampling 4 chains, 0 divergences] |--------------------------------| 0.24% [39/16000 08:04<55:01:52 Sampling 4 chains, 0 divergences] |--------------------------------| 0.25% [40/16000 08:05<53:49:27 Sampling 4 chains, 0 divergences] |--------------------------------| 0.26% [41/16000 08:06<52:37:37 Sampling 4 chains, 0 divergences] |--------------------------------| 0.26% [42/16000 08:11<51:49:39 Sampling 4 chains, 0 divergences] |--------------------------------| 0.27% [43/16000 08:22<51:48:40 Sampling 4 chains, 0 divergences] |--------------------------------| 0.28% [44/16000 08:37<52:04:47 Sampling 4 chains, 0 divergences] |--------------------------------| 0.28% [45/16000 08:38<51:03:35 Sampling 4 chains, 0 divergences] |--------------------------------| 0.29% [46/16000 08:56<51:38:57 Sampling 4 chains, 0 divergences] |--------------------------------| 0.29% [47/16000 09:02<51:11:18 Sampling 4 chains, 0 divergences] |--------------------------------| 0.30% [48/16000 09:13<51:05:41 Sampling 4 chains, 0 divergences] |--------------------------------| 0.31% [49/16000 09:16<50:21:01 Sampling 4 chains, 0 divergences] |--------------------------------| 0.31% [50/16000 09:34<50:53:40 Sampling 4 chains, 0 divergences] |--------------------------------| 0.32% [51/16000 09:44<50:44:55 Sampling 4 chains, 0 divergences] |--------------------------------| 0.33% [52/16000 09:47<50:01:37 Sampling 4 chains, 0 divergences] |--------------------------------| 0.33% [53/16000 09:56<49:51:06 Sampling 4 chains, 0 divergences] |--------------------------------| 0.34% [54/16000 10:11<50:08:37 Sampling 4 chains, 0 divergences] |--------------------------------| 0.34% [55/16000 10:15<49:36:13 Sampling 4 chains, 0 divergences] |--------------------------------| 0.35% [56/16000 10:27<49:38:14 Sampling 4 chains, 0 divergences] |--------------------------------| 0.36% [57/16000 10:32<49:08:06 Sampling 4 chains, 0 divergences] |--------------------------------| 0.36% [58/16000 10:53<49:51:42 Sampling 4 chains, 0 divergences] |--------------------------------| 0.37% [59/16000 10:55<49:13:44 Sampling 4 chains, 0 divergences] |--------------------------------| 0.38% [60/16000 11:01<48:51:07 Sampling 4 chains, 0 divergences] |--------------------------------| 0.38% [61/16000 11:17<49:09:20 Sampling 4 chains, 0 divergences] |--------------------------------| 0.39% [62/16000 11:18<48:26:55 Sampling 4 chains, 0 divergences] |--------------------------------| 0.39% [63/16000 11:33<48:42:09 Sampling 4 chains, 0 divergences] |--------------------------------| 0.40% [64/16000 11:40<48:28:19 Sampling 4 chains, 0 divergences] |--------------------------------| 0.41% [65/16000 11:47<48:12:04 Sampling 4 chains, 0 divergences] |--------------------------------| 0.41% [66/16000 11:52<47:48:05 Sampling 4 chains, 0 divergences] |--------------------------------| 0.42% [67/16000 12:11<48:20:57 Sampling 4 chains, 0 divergences] |--------------------------------| 0.42% [68/16000 12:16<47:55:07 Sampling 4 chains, 0 divergences] |--------------------------------| 0.43% [69/16000 12:20<47:30:15 Sampling 4 chains, 0 divergences] |--------------------------------| 0.44% [70/16000 12:35<47:46:52 Sampling 4 chains, 0 divergences] |--------------------------------| 0.44% [71/16000 12:46<47:46:36 Sampling 4 chains, 0 divergences] |--------------------------------| 0.45% [72/16000 12:53<47:32:08 Sampling 4 chains, 0 divergences] |--------------------------------| 0.46% [73/16000 13:00<47:17:52 Sampling 4 chains, 0 divergences] |--------------------------------| 0.46% [74/16000 13:13<47:26:24 Sampling 4 chains, 0 divergences] |--------------------------------| 0.47% [75/16000 13:14<46:51:31 Sampling 4 chains, 0 divergences] |--------------------------------| 0.47% [76/16000 13:41<47:47:45 Sampling 4 chains, 0 divergences] |--------------------------------| 0.48% [77/16000 13:43<47:19:35 Sampling 4 chains, 0 divergences] |--------------------------------| 0.49% [78/16000 13:47<46:53:44 Sampling 4 chains, 0 divergences] |--------------------------------| 0.49% [79/16000 13:53<46:39:24 Sampling 4 chains, 0 divergences] |--------------------------------| 0.50% [80/16000 14:15<47:17:49 Sampling 4 chains, 0 divergences] |--------------------------------| 0.51% [81/16000 14:17<46:47:11 Sampling 4 chains, 0 divergences] |--------------------------------| 0.51% [82/16000 14:28<46:48:32 Sampling 4 chains, 0 divergences] |--------------------------------| 0.52% [83/16000 14:40<46:54:35 Sampling 4 chains, 0 divergences] |--------------------------------| 0.53% [84/16000 14:41<46:22:22 Sampling 4 chains, 0 divergences] |--------------------------------| 0.53% [85/16000 14:46<46:06:08 Sampling 4 chains, 0 divergences] |--------------------------------| 0.54% [86/16000 14:52<45:51:38 Sampling 4 chains, 0 divergences] |--------------------------------| 0.54% [87/16000 15:17<46:37:27 Sampling 4 chains, 0 divergences] |--------------------------------| 0.55% [88/16000 15:21<46:16:10 Sampling 4 chains, 0 divergences] |--------------------------------| 0.56% [89/16000 15:22<45:48:24 Sampling 4 chains, 0 divergences] |--------------------------------| 0.56% [90/16000 15:28<45:36:25 Sampling 4 chains, 0 divergences] |--------------------------------| 0.57% [91/16000 15:44<45:52:30 Sampling 4 chains, 0 divergences] |--------------------------------| 0.57% [92/16000 15:47<45:31:12 Sampling 4 chains, 0 divergences] |--------------------------------| 0.58% [93/16000 15:54<45:20:36 Sampling 4 chains, 0 divergences] |--------------------------------| 0.59% [94/16000 16:01<45:12:22 Sampling 4 chains, 0 divergences] |--------------------------------| 0.59% [95/16000 16:13<45:15:03 Sampling 4 chains, 0 divergences] |--------------------------------| 0.60% [96/16000 16:18<45:01:44 Sampling 4 chains, 0 divergences] |--------------------------------| 0.61% [97/16000 16:21<44:40:36 Sampling 4 chains, 0 divergences] |--------------------------------| 0.61% [98/16000 16:45<45:18:23 Sampling 4 chains, 0 divergences] |--------------------------------| 0.62% [99/16000 16:45<44:51:10 Sampling 4 chains, 0 divergences] |-------------------------------| 0.62% [100/16000 16:47<44:31:01 Sampling 4 chains, 0 divergences] |-------------------------------| 0.63% [101/16000 16:48<44:05:48 Sampling 4 chains, 0 divergences]